File: BasicInputOutput/Java/ArithmeticOperation.java
import java.util.Scanner;

class ArithmeticOperation{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter the Two Value : ");
        double num1 = sc.nextDouble();
        double num2 = sc.nextDouble();

        System.out.println("Addition A + B : "+(num1+num2));
        System.out.println("Subtraction A - B : "+(num1-num2));
        System.out.println("Multiply A * B : "+(num1*num2));
        System.out.println("Divsion A / B : "+(num1/num2));
        System.out.println("Modulus A % B : "+(num1%num2));
        System.out.println("Inc Dec:- A++ --B --A B++ : "+ num1++ +" "+ + --num2 +" "+ --num1 +" "+ num2++);
    }
}

File: BasicInputOutput/C/ArithmeticOperation.c
#include <stdio.h>

int main() {
    double num1, num2;
    printf("Enter the Two Values: ");
    scanf("%lf %lf", &num1, &num2);

    printf("Addition A + B: %.2lf\n", num1 + num2);
    printf("Subtraction A - B: %.2lf\n", num1 - num2);
    printf("Multiply A * B: %.2lf\n", num1 * num2);
    printf("Division A / B: %.2lf\n", num1 / num2);
    printf("Modulus A %% B: %.2lf\n", fmod(num1, num2));
    
    printf("Inc Dec: A++ --B --A B++: %.2lf %.2lf %.2lf %.2lf\n", 
           num1++, --num2, --num1, num2++);

    return 0;
}
